modules/cudaoptflow/samples/nvidia_optical_flow.cpp

```diff
@@ -198,6 +198,7 @@ bool parseROI(std::string ROIFileName, std::vector<Rect>& roiData)
 
 int main(int argc, char **argv)
 {
+#if defined HAVE_NVIDIA_OPTFLOW
```

Comment:
Samples should not depend on such defines.
API calls will throw an exception.

Please revert compilation guards from samples.

Reply:
Removed. thank you